test/csharp/test_CorePsPlatform.cs

```diff
@@ -61,7 +62,7 @@ public static void TestGetMachineName()
             }
         }
 
-        [Fact(Skip="Bad arguments for macOS")]
+        [Fact]
```

Comment:
i believe that this is still not good for MacOS. `hostname --fqdn` is an error. Will this run on Mac?